float32 linear
float32 angular

string flag
float64 theta
